### PR TITLE
Handle trove events for dsproxy troves.

### LINKED ISSUE #3964: Liquity: Missing movements for DSProxy Troves
## Liquity: Missing movements for DSProxy Troves

Trove operations show up fine for regular troves, but not for troves owned through a DSProxy (e.g. created on DeFiSaver)

Screenshot_20220118_172422

## Motivation

It feels incomplete and inconsistent.

### PR DESCRIPTION
**Pull request by yabirgb**
Closes #3964

### CODE REVIEW

File: docs/changelog.rst [hunk 1]
```diff
@@ -4,6 +4,7 @@ Changelog
 
 * :bug:`-` Users will now be properly prompted to restart the application after the auto-updater downloads the update.
 * :bug:`3943` Users will now be able to properly add multiple accounts on Avalance even if they exist on Ethereum.
+* :bug:`3964` DSProxy Troves will now see their events properly listed.
```
Comment (LefterisJP) [inline docs/changelog.rst:7]:
```suggestion
* :bug:`3964` Liquity Troves managed by a DSProxy will now see their events properly listed.
```

**Review by LefterisJP — COMMENTED** [review]
I guess you tested this. Didn't you say this also needs a frontend change?

**Comment by yabirgb** [comment]
Yes, tested it! The frontend comment is a different issue. Not related to this PR. Is something that I detected while testing the events with the DSProxy account

**Comment by LefterisJP** [comment]
> Yes, tested it! The frontend comment is a different issue. Not related to this PR. Is something that I detected while testing the events with the DSProxy account

Yes but if DSProxy account breaks the frontend display of troves it's very much related, no?

**Review by LefterisJP — APPROVED** [review]
LGTM if you tested and works fine. Perhaps you can also provide a test binary to the user to confirm fix on their side.

**Pull request merged**